docs/kyma/04-02-local-installation.md

```diff
@@ -110,59 +126,22 @@ To learn how to test Kyma, see [this](#details-testing-kyma) document.
 Use the `minikube.sh` script to restart the Minikube cluster without reinstalling Kyma. Follow these steps to stop and restart your cluster:
 
 1. Stop the Minikube cluster with Kyma installed. Run:
-  ```
-  minikube stop
-  ```
+   ```
+   minikube stop
+   ```
 2. Restart the cluster without reinstalling Kyma. Run:
-    <div tabs>
-      <details>
-      <summary>
-      MacOS
-      </summary>
-
-      ```
-      ./scripts/minikube.sh --domain "kyma.local" --vm-driver hyperkit
-      ```
-      </details>
-      <details>
-      <summary>
-      Linux
-      </summary>
-
-      ```
-      ./scripts/minikube.sh --domain "kyma.local" --vm-driver virtualbox
-      ```
-      </details>
-    </div>
-
-The script discovers that a Minikube cluster is initialized and asks if you want to delete it. Answering `no` causes the script to start the Minikube cluster and restarts all of the previously installed components. Even though this procedure takes some time, it is faster than a clean installation as you don't download all of the required Docker images.
+   ```bash
+   kyma provision minikube
+   ```
+
+The Kyma CLI discovers that a minikube cluster is initialized and asks if you want to delete it. Answering `no` causes the Kyma CLI to start the minikube cluster and restarts all of the previously installed components. Even though this procedure takes some time, it is faster than a clean installation as you don't download all of the required Docker images.
 
 To verify that the restart is successful, run this command and check if all Pods have the `RUNNING` status:
 
 ```
 kubectl get pods --all-namespaces
 ```
 
-## Enable Horizontal Pod Autoscaler (HPA)
```

Comment:
With the Kyma CLI, is it enabled automatically during installation? How can you enable it?